emplace: 0.4.4 -> 1.0.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/tversteeg/emplace/releases.


meta.description for emplace is: "Mirror installed software on multiple machines"

meta.homepage for emplace is: "https://github.com/tversteeg/emplace"

meta.changelog for emplace is: ""


###### Updates performed
- Rust version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/tversteeg/emplace/releases/tag/v1.0.0)

- [Compare changes on GitHub](https://github.com/tversteeg/emplace/compare/v0.4.4...v1.0.0)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 1.0.0 with grep in /nix/store/0di3xm0h7r9np6a295ximv6033jrcaz1-emplace-1.0.0
- directory tree listing: https://gist.github.com/0ec185dc3f2ed9ea4f8c7fd5b413a4c1

- du listing: https://gist.github.com/36abbbe13f5858b71d144e612b375ee1

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
1 x86_64-darwin rebuild(s)
1 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
emplace
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/0di3xm0h7r9np6a295ximv6033jrcaz1-emplace-1.0.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A emplace https://github.com/r-ryantm/nixpkgs/archive/80185972d92ba9bcbe35aa03b09e2fb2df669afa.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/0di3xm0h7r9np6a295ximv6033jrcaz1-emplace-1.0.0
ls -la /nix/store/0di3xm0h7r9np6a295ximv6033jrcaz1-emplace-1.0.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>emplace</li>
  </ul>
</details>

---

###### Maintainer pings

cc @Br1ght0ne for testing.

r-ryantm committed:
emplace: 0.4.4 -> 1.0.0

r-rmcgibbo commented:
Result of `nixpkgs-review pr 112787` at 80185972 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/2952766438434d63c5fe81fc12e43e44">emplace</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`agpl3` is a deprecated license, check if project uses `agpl3Plus` or `agpl3Only` and change `meta.license` accordingly.

Near pkgs/tools/package-management/emplace/default.nix:19:5:
```
   |
19 |     license = licenses.agpl3;
   |     ^
```
</li>
  </ul>
</details>

Br1ght0ne committed:
emplace: change meta.license to agpl3Plus

Taken from https://github.com/tversteeg/emplace/blob/master/Cargo.toml#L9.

Suggested-by: Robot Robert <r-rmcgibbo@users.noreply.github.com>

Br1ght0ne reviewed (approved):
Diff LGTM. Applied the suggestion for `agpl3Plus`.
